independent-projects/tools/devtools-common/src/main/java/io/quarkus/devtools/commands/handlers/CreateProjectCommandHandler.java

```diff
@@ -137,11 +182,12 @@ public QuarkusCommandOutcome execute(QuarkusCommandInvocation invocation) throws
                     .example(invocation.getValue(EXAMPLE))
                     .noCode(invocation.getValue(NO_CODE, false))
                     .addCodestarts(invocation.getValue(EXTRA_CODESTARTS, Collections.emptySet()))
-                    .noBuildToolWrapper(invocation.getValue(NO_BUILDTOOL_WRAPPER, false))
+                    .noBuildToolWrapper(invocation.getValue(NO_BUILDTOOL_WRAPPER, parent != null))
```

Comment:
I don't know that this is the best assumption, though I understand why you're doing it this way. We have a separate issue open for detecting wrappers that would interfere here.

Reply:
This based on the issue description https://github.com/quarkusio/quarkus/issues/12936#issue-729345977
If it is not applicable I'll leave it as false

Reply:
@ebullient why would it interfere since we still use the invocation value? Cristian change is only affecting the default behavior..

Reply:
@cristian-com after giving it a try, it seems not be working since this default is never* used.

* because we have another default overriding it in the caller..


Maybe we shouldn't pass any default in the callers when not expressly set by the user